.github/dependabot.yaml

```diff
@@ -6,6 +6,19 @@ updates:
   - dependency-type: "all"
   schedule:
     interval: daily
```

Comment:
Should we change the interval? Because the groups will very likely update in different days.